src/view/DialogNew.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {FormEvent} from "react";
+import {Innholdstittel, Normaltekst} from "nav-frontend-typografi";
+import {Input, Textarea} from "nav-frontend-skjema";
+import {Hovedknapp} from "nav-frontend-knapper";
+import useFieldState from "../utils/useFieldState";
+
+import './Dialog.less';
+
+function validerTema(tema: string): string | null {
+    if (tema.length === 0) {
+        return "Tema må ha innhold.";
+    } else if (!tema.replace(/\s/g, '').length) {
+        return "Tema må ha lesbart innhold.";
+    } else {
+        return null;
+    }
+}
+function validerMelding(melding: string): string | null {
+    if (melding.length === 0) {
+        return "Melding må ha innhold.";
+    } else if (!melding.replace(/\s/g, '').length) {
+        return "Meldingen må ha lesbart innhold.";
+    } else {
+        return null;
+    }
+}
+
+
+export function DialogNew() {
+
+    const tema = useFieldState('', validerTema);
+    const melding = useFieldState('', validerMelding);
+
+    function handleSubmit(event: FormEvent) {
+        event.preventDefault();
+        tema.validate();
+        melding.validate();
+
+        if (tema.input.feil === undefined && melding.input.feil === undefined) {
+            fetch('/veilarbdialog/api/dialog/ny', {
```

Comment:
bedre å flytte post logikken ut i en fetch util fil, for så ta den i bruk her. blir lettere å lese

type: 
https://github.com/navikt/behovsvurdering/blob/master/src/api/api.ts

Reply:
tas til etterretning i FO-2372 hvor DialogNew.tsx  hører til.

Reply:
Vi allerede har en fetchData ifm get, mens dette er ifm post. bør vi kombinere dem?